src/popup/components/TransactionOverview.vue

```diff
@@ -185,8 +182,28 @@ export default defineComponent({
             },
           };
         }
+        case Tag.GaMetaTx: {
+          if (innerTxType.value === Tag.SpendTx) {
+            return {
+              sender: {
+                address: senderId,
+                name: getPreferredName.value(senderId),
+                url: aeScan.prepareUrlByHash(senderId),
+                label: t('transaction.overview.accountAddress'),
+              },
+              recipient: {
+                address: recipientId,
+                name: name.value || getPreferredName.value(recipientId),
+                url: aeScan.prepareUrlByHash(recipientId),
+                label: t('transaction.overview.accountAddress'),
+              },
+              title: t('transaction.type.spendTx'),
+            };
+          }
+        }
+        // eslint-disable-next-line no-fallthrough
```

Comment:
Shouldn't we care about other innerTxTypes?

Reply:
No, We support only SpendTx in multisig vaults